__tests__/app.test.js

```diff
@@ -148,4 +147,75 @@ describe('GET /api/articles/:article_id/comments task 6',() => {
             expect(res.body.comments).toEqual([])
         });
     });
+});
+
+//7
+describe('POST /api/articles/:article_id/comments task 6',() => {
+    test('responds with status code 201, and posts comment with appropriate data types',() => {
+        const reqarticle_id = 1;
```

Comment:
Whilst descriptive - it might be easier to read this variable name if it was camelCased